portfolio/src/main/java/com/google/sps/utils/CommentDataStore.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.utils;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.repackaged.com.google.datastore.v1.Datastore;
+import com.google.sps.data.Comment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Provides the service to interact with Datastore to store or load comment. */
+public class CommentDataStore {
```

Comment:
Why not let CommentDataStore implements ObjectDataStore<Comment>?

Reply:
Since the methods are static... seems cannot implement static method from interface?

Reply:
Fixed; make it as instance method